### PR TITLE
[TravisCI] Split CXX/Python tests on Linux

### PR DESCRIPTION
**Pull request by jschueller**
Latests PRs seem to timeout because of them

### CODE REVIEW

**Comment by dbarbier** [comment]
I disagree, it is always good to keep an eye on C++ tests.

**Comment by jschueller** [comment]
I want to check them too, but the fact is that we often exceed the 50 minutes limit.

**Comment by regislebrun** [comment]
How can we get rid of this 50 minutes limit? Skipping tests because they cost too much does not make sense to me! Is it possible to start several compilation targets, one focusing on the C++ library (with tests) and the other on the Python binding (with tests+docstring tests+notebooks+...)?

**Comment by jschueller** [comment]
Ci tests should always succeed. But instead of skipping tests I can add another job that runs only the cxx tests.

**Comment by dbarbier** [comment]
Maybe we could now reenable gcc on Linux? ;-)

**Comment by jschueller** [comment]
@dbarbier Ok

**Comment by regislebrun** [comment]
+1

File: .travis.yml [hunk 1]
```diff
@@ -22,7 +22,11 @@ addons:
 matrix:
   include:
     - os: linux
-      compiler: clang
+      compiler: gcc
+      env: BUILD_PYTHON=ON
```
Comment (sofianehaddad) [inline .travis.yml]: We could use both 'gcc' and 'clang' with two specific env ?

Comment (jschueller) [reply .travis.yml]: Ok, whatever

**Comment by jschueller** [comment]
Seems the gcc jobs are less reliable: I got failures yesterday.

**Pull request merged**